test/ui-test/walkthroughUiTest.ts

```diff
@@ -51,10 +61,15 @@ describe("Check walkthroughs, elements and associated commands", async () => {
       await commandInput.setText(`${walkthroughName}`);
       await commandInput.confirm();
 
-      await sleep(1000);
-
       // Select the editor window
-      const welcomeTab = await editorView.getTabByTitle("Walkthrough: Ansible");
+      const welcomeTab = await VSBrowser.instance.driver.wait(
+        async () => {
+          return await editorView.getTabByTitle("Walkthrough: Ansible");
+        },
+        undefined,
+        "Timed out waiting for walkthrough tab to open",
```

Comment:
Where is the timeout value? While removing the sleep is what we want, we still need a a maximum wait time and that is not visible inside the example. I suspect that the 'undefined' parameter might be the one but that hides that critical value.

I think that we should never rely on the default and make all the timeouts explicit. OK if we define them as variables and reuse them, but still not relying on 'undefined' as this is opaque and does not inform the 'random' developer looking at the code about its existence.